.pre-commit-config.yaml

```diff
@@ -71,6 +71,7 @@ repos:
     hooks:
       - id: codespell
         additional_dependencies: ['tomli']
+        args: [-L, lama]
```

Comment:
This is not really a scalable solution. I propose to use word lists (` -I FILE, --ignore-words=FILE` options) at the project level.

Reply:
Depending on the technical details of the above issue, this one will also be redundant or come up earlier there.